docs/pages/database-access/guides/rds.mdx

```diff
@@ -344,10 +344,12 @@ registered databases:
 ```code
 $ tsh login --proxy=teleport.example.com --user=alice
 $ tsh db ls
-# Name         Description                 Labels
-# ------------ --------------------------- --------
-# postgres-rds RDS instance in us-west-1   ...
-# aurora-mysql Aurora cluster in us-west-1 ...
+# Name                           Description                                   Labels
```

Comment:
@greedy52 Let's also add a note somewhere in this guide that explains that Teleport will discover your RDS primary endpoint, reader endpoint and custom endpoints and the way they will be named.

Reply:
not very good at documentation =p. I've updated some paragraphs and notes for view.

Reply:
@klizhentas please review the docs when you have a chance. Thanks